common/script/ops/scoreTask.js

```diff
@@ -141,27 +157,34 @@ function _changeTaskValue (user, task, direction, times, cron) {
 
   // If multiple days have passed, multiply times days missed
   _.times(times, () => {
-    // Each iteration calculate the nextDelta, which is then accumulated in the total delta.
-    let nextDelta = !cron && direction === 'down' ? _calculateReverseDelta(task, direction) : _calculateDelta(task, direction, cron);
-
-    if (user.preferences.automaticAllocation === true && user.preferences.allocationMode === 'taskbased' && !(task.type === 'todo' && direction === 'down')) {
-      user.stats.training[task.attribute] += nextDelta;
-    }
+    // Each iteration calculates a change in task value, which is then accumulated in the total delta.
+    addToDelta += !cron && direction === 'down' ? _calculateReverseDelta(task) : _calculateDelta(task, direction, cron);
```

Comment:
since _calculateDelta has always direction === 'up', the `direction` parameter should be removed?